modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -26,8 +26,53 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the installation directory that
 contains the `install-config.yaml` file you created.
 
-. Create the machine set manifests for the worker nodes in your Local Zone.
+. Set the default Maximum Transmission Unit (MTU) according to the network plugin:
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The cluster network MTU must be always less than the EC2 MTU to account the overhead. The specific overhead is determined by your network plugin, for example:
```

Comment:
Not sure if this should be "to account _for_ the overhead".

Reply:
Done, thanks!